src/main/java/core/selenium/WebDriverManager.java

```diff
@@ -5,31 +5,54 @@
 
 import java.util.concurrent.TimeUnit;
 
+/**
+ * WebDriverManager class, this class is responsible for creating an instance of a browser.
+ *
+ * @author Maday Alcala
+ * @version 0.0.1
+ */
 public final class WebDriverManager {
-    private WebDriverConfig webDriverConfig = WebDriverConfig.getInstance();
-    private WebDriver webDriver;
+    private static WebDriverConfig webDriverConfig = WebDriverConfig.getInstance();
+    private static WebDriver webDriver;
     private static WebDriverManager instance = null;
```

Comment:
WebDriverManager is a singleton not need to declare or instantiate.

Reply:
I can't remove that statement from the variable 'instance' as it is being used to verify that no more than one instance is created, according to the singleton pattern.

Reply:
Solved.